v1.0/monitor-cockroachdb-with-prometheus.md

```diff
@@ -36,19 +36,20 @@ Make sure you have already started a CockroachDB cluster, either [locally](start
 ## Step 2. Configure Prometheus
 
 1. Download the starter [Prometheus configuration file](https://github.com/cockroachdb/cockroach/blob/master/monitoring/prometheus.yml) and [aggregation rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/aggregation.rules) for CockroachDB:
-   
+
     ~~~ shell
     # Configuration file:
     $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/prometheus.yml \
-    -O prometheus.y    
+    -O prometheus.y
```

Comment:
Seems I never noticed this before. This should be `-O prometheus.yml`, that's what we use later on.